src/metabase/api/pulse_channel_recipient.clj

```diff
@@ -0,0 +1,18 @@
+(ns metabase.api.pulse-channel-recipient
+  "/api/pulse-channel-recipient endpoints."
+  (:require [compojure.core :refer [DELETE]]
+            [metabase.api.common :as api]
+            [metabase.events :as events]
+            [metabase.models.pulse-channel-recipient :refer [PulseChannelRecipient]]
+            [toucan.db :as db]))
+
+(api/defendpoint DELETE "/:id"
+  "Delete a Pulse. (DEPRECATED -- don't delete a Pulse anymore -- archive it instead.)"
+  [id]
+  (api/let-404 [pcr (PulseChannelRecipient id)]
+               (api/write-check PulseChannelRecipient id)
+               (db/delete! PulseChannelRecipient :id id)
+               (events/publish-event! :pulse-channel-recipient-delete (assoc pcr :actor_id api/*current-user-id*)))
+  api/generic-204-no-content)
```

Comment:
Docstring seems wrong

Also it seems like something like a `DELETE /api/pulse/:pulse-id/membership/:user-id` endpoint would be more useful than this endpoint so you don't have to know the ID of the PulseChannelRecipient row itself. As far as clients are concerned they shouldn't need to know that the PulseChannelRecipient model exists at all

Reply:
`let-404` *should* have `:style/indent 1` metadata, e.g. it should have been indented just like a `let` form. I will have to double-check that

Reply:
Actually if it's used for removing your own "subscription" to the Pulse, `DELETE /api/pulse/:pulse-id/subscription` might make more sense (because you can still edit the whole list of Users via the `PUT` endpoint, if you have write permissions). See [`DELETE /api/card/:id/favorite`](https://github.com/metabase/metabase/blob/master/src/metabase/api/card.clj#L487).

Thoughts @tlrobinson ?